knip.js

```diff
@@ -0,0 +1,104 @@
+// @ts-check
+const testEntry = 'test/**/*.test.js';
+
+/** @type {import('knip').KnipConfig} */
+export default {
+	ignore: ['**/test/**/{fixtures,_temp-fixtures}/**', '.github/scripts/**'],
+	tags: ['-lintignore'],
+	ignoreWorkspaces: [
+		'examples/**',
+		'**/{test,e2e}/**/{fixtures,_temp-fixtures}/**',
+		'benchmark/**',
+	],
+	workspaces: {
+		'.': {
+			ignoreDependencies: ['@astrojs/check'],
```

Comment:
It think it could be beneficial add a comment to all `ignoreDependencies` and explain why